test/Macros/parsing.swift

```diff
@@ -56,3 +56,6 @@ macro am2() -> Void
 #m1 + 1
 // expected-warning @-1 {{result of operator '+' is unused}}
 // expected-error @-2 {{external macro implementation type 'A.M1' could not be found for macro 'm1()'}}
+
+_ = #parsing.m1
```

Comment:
This is syntactically ambiguous between
* `#parsing` found an expression macro and the resulting value has a member `m1`, and
* `m1` is a macro in the module `parsing`

Reply:
For a very specific example: `print(#file.dropFirst())` is well-formed today.